mobile/test/common/integration/base_client_integration_test.cc

```diff
@@ -203,16 +203,6 @@ void BaseClientIntegrationTest::createEnvoy() {
     }
   }
 
-  if (override_builder_config_) {
```

Comment:
I don't think `override_builder_config_` is used anywhere anymore now after this PR, so we can remove it from the header file (https://github.com/envoyproxy/envoy/blob/6f95c27ef7a65f74a0a44783068690c7bf10f993/mobile/test/common/integration/base_client_integration_test.h#L80)

Reply:
This is already removed.